x/concentrated-liquidity/position_test.go

```diff
@@ -119,6 +170,21 @@ func (s *KeeperTestSuite) TestInitOrUpdatePosition() {
 			if test.expectedErr != nil {
 				s.Require().Error(err)
 				s.Require().ErrorContains(err, test.expectedErr.Error())
+
+				// If the error is due to a nonexistent pool, we exit before pool-level checks
+				if test.param.poolId == invalidPoolId {
+					return
+				}
```

Comment:
Why don't we still check that uptime accumulators weren't updated?

Reply:
Since if the pool doesn't exist, neither does the uptime accumulators so there's nothing to check! If you mean checking that the uptime accumulators don't exist then that doesn't belong at this level of abstraction and would be redundant with accum creation/pool creation tests